src/inc/internal/BundleManifestWriter.hpp

```diff
@@ -0,0 +1,54 @@
+//
+//  Copyright (C) 2019 Microsoft.  All rights reserved.
+//  See LICENSE file in the project root for full license information.
+// 
+#pragma once
+
+#include "XmlWriter.hpp"
+#include "ComHelper.hpp"
+#include "Exceptions.hpp"
+#include "UnicodeConversion.hpp"
+
+#include <vector>
+
+namespace MSIX {
+
+    enum ElementWriterState
+    {
+        Uninitialized = 0,
+        BundleManifestStarted = 1,
+        PackagesAdded = 2,
+        OptionalBundlesAdded = 3,
+        BundleManifestEnded = 4,
+    };

```

Comment:
try using scoped enumerations when possible

Reply:
Removed this enum since it was not being used anywhere as of now.